vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/ApiRegisterUser.java

```diff
@@ -0,0 +1,43 @@
+package fr.insalyon.creatis.vip.api.business;
+
+import fr.insalyon.creatis.vip.api.exception.ApiException;
+import fr.insalyon.creatis.vip.core.client.bean.User;
+import fr.insalyon.creatis.vip.core.server.business.BusinessException;
+import fr.insalyon.creatis.vip.core.server.business.ConfigurationBusiness;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author khalilkes service to signup a user in VIP
+ */
+@Service
+public class ApiRegisterUser {
```

Comment:
`ApiUserBusiness` wourd be a better class name

Reply:
agreed